lopezMidterm/src/reducers/cartReducer.ts

```diff
@@ -0,0 +1,46 @@
+import { ADD_TO_CART, REMOVE_FROM_CART, CLEAR_CART, CartAction, CartItem } from '../types/cartTypes';
+
+interface CartState {
+  items: CartItem[];
+}
+
+const initialState: CartState = {
+  items: [],
```

Comment:
You could use `items.length` to display the number of items in your cart in your header. The main reason that Redux exists is for global state. By showing the number of cart items, we are demonstrating that we can share state across multiple screens.